Gemfile

```diff
@@ -2,3 +2,23 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in activeadmin_settings_cached.gemspec
 gemspec
+
+group :development do
+  gem 'rubocop'
+end
+
+group :test do
+  gem 'rails', '~> 4.2.5'
+  gem 'turbolinks'
+  gem 'rspec-rails'
+  gem 'rails-settings-cached', github: 'merqlove/rails-settings-cached'
+  gem 'activeadmin', github: 'activeadmin/activeadmin', branch: 'ab1d8d1a4a1af748ae7009d9d725193d780d0059'
+  gem 'coveralls', require: false
+  gem 'sass-rails'
+  gem 'sqlite3'
+  gem 'launchy'
+  gem 'database_cleaner'
+  gem 'capybara'
+  gem 'selenium-webdriver'
+  gem 'poltergeist'
```

Comment:
Github dep’s will not work from gemspec.
--
Best regards,
Alexander Merkulov

E-mail: sasha@merqlove.ru
Tel.:  +7(925) 278-44-98
Skype:  merqlove



> On 10 янв. 2016 г., at 19:27, Semyon Pupkov <notifications@github.com> wrote:
> 
> In Gemfile <https://github.com/artofhuman/activeadmin_settings_cached/pull/12#discussion_r49277893>:
> 
> > +end
> > +
> > +group :test do
> > +  gem 'rails', '~> 4.2.5'
> > +  gem 'turbolinks'
> > +  gem 'rspec-rails'
> > +  gem 'rails-settings-cached', github: 'merqlove/rails-settings-cached'
> > +  gem 'activeadmin', github: 'activeadmin/activeadmin', branch: 'ab1d8d1a4a1af748ae7009d9d725193d780d0059'
> > +  gem 'coveralls', require: false
> > +  gem 'sass-rails'
> > +  gem 'sqlite3'
> > +  gem 'launchy'
> > +  gem 'database_cleaner'
> > +  gem 'capybara'
> > +  gem 'selenium-webdriver'
> > +  gem 'poltergeist'
> Ithink this should be placed in gemspec file. spec.add_development_dependency "capybara" etc
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/artofhuman/activeadmin_settings_cached/pull/12/files#r49277893>.
>